mission/functions/core/ui/taskroster/fn_tr_teamInfo_listboxSelect.sqf

```diff
@@ -0,0 +1,15 @@
+// TODO
+
+disableSerialization;
+#include "..\..\..\..\config\ui\ui_def_base.inc"
+
+private _index = lnbCurSelRow VN_TR_SHOWTEAMINFO_RHS_CHANGETEAM_LISTBOX_CTRL;
+private _selectedTeam = VN_TR_SHOWTEAMINFO_RHS_CHANGETEAM_LISTBOX_CTRL lnbData [_index, 0];
+["changeteam", [_selectedTeam]] call para_c_fnc_call_on_server;
```

Comment:
to actually change team, we remoteExec over to the change team logic on the server, which then remote execs back with all the player local changes once the overview update function is run.